ServerSide/yoty/Core/Managers/Bids/BidsManager.cs

```diff
@@ -84,100 +84,100 @@ public async Task<Response<SupplierProposalDTO>> AddSupplierProposal(SupplierPro
 
         public async Task<Response<BidDTO>> CreateNewBid(NewBidRequst bidRequest)
         {
-            BidEntity bidEnitity = _mapper.Map<BidEntity>(bidRequest);
+            BidEntity bidEntity = _mapper.Map<BidEntity>(bidRequest);
             //TODO is this the time we want? (or global).
-            bidEnitity.CreationDate = DateTime.Now;
-            bidEnitity.Id = Guid.NewGuid().ToString();
-            bidEnitity.UnitsCounter = 0;
-            bidEnitity.PotenialSuplliersCounter = 0;
-            bidEnitity.Product.Id = Guid.NewGuid().ToString();
+            bidEntity.CreationDate = DateTime.Now;
+            bidEntity.Id = Guid.NewGuid().ToString();
+            bidEntity.UnitsCounter = 0;
+            bidEntity.PotenialSuplliersCounter = 0;
```

Comment:
since its int its already initialized to 0 in the mapping, remove.

Reply:
Done